packages/salesforcedx-vscode-apex/src/embeddedSoql/soqlCompletion.ts

```diff
@@ -14,9 +14,9 @@ import {
   Uri,
   workspace
 } from 'vscode';
+import ProtocolCompletionItem from 'vscode-languageclient/lib/common/protocolCompletionItem';
```

Comment:
This seems ...unusal.  Do you understand why we're pulling in a default export from deep in the vslsp lib? I wonder if `CompletionItem` can be used instead.

Reply:
@gbockus-sf At this moment I do not understand the reason to "dip into" the non-public area of the client.

Reply:
Word.  I can create a WI to follow up on this.